package/tests/testthat/_snaps/mhl01/markdown-snaps.md

```diff
@@ -0,0 +1,20 @@
+# lsting
+
+    Code
+      print(data_snap[[i]])
+    Output
+      # A tibble: 1,934 x 11
+         ID     ASR   TRT01A MHBODSYS MHDECOD TRTSDTM ASTDTM ASTDY AENDTM AENDY ATIREL
```

Comment:
I think this may need to be discussed with Pawel later in the week. @pawelru @edelarua fyi

Reply:
you mean to print the whole table?

Reply:
yes we should select and print relevant lines and not only the first ones

Reply:
I agree. We need to either modify all `print()` statements to print all cols and rows or use [`pillar` options](https://pillar.r-lib.org/reference/pillar_options.html) for tibble printing during creating snapshots (assuming that all examples are of tibble class). @vedhav can you please have a look and create an issue?